types/mapbox__s3urls/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for mapbox__s3urls 1.5
+// Project: https://github.com/mapbox/s3urls
+// Definitions by: Sebastian Vera <https://github.com/sebastianvera>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// tslint:disable-next-line no-single-declare-module
+declare module "@mapbox/s3urls" {
```

Comment:
This module declaration syntax is good to use if you are declaring multiple modules in a single file. Using top level exports is better in this case since this declaration file is for just one module. [Here is a template that you can use.](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html)

Reply:
@PranavSenthilnathan even if is a scoped module? I tried with that template but autocomplete didn't work, this syntax is the only way it works, maybe am I doing something wrong?

Reply:
@PranavSenthilnathan can you please confirm if I really need to use top-level imports, even though it's a scoped library `@mapbox/s3urls`?

Thanks!

Reply:
Sorry, you're correct about that!